composer.json

```diff
@@ -14,7 +14,7 @@
 	],
 	"require": {
 		"php": ">=5.6",
-		"cakephp/orm": "^3.4",
+		"cakephp/orm": "^3.6",
```

Comment:
I will only accept ^3.5 at this time. Please make sure you use anything compatible for this, and check travis matrix.
People will have to turn off deprecations for now.

Reply:
Refs https://github.com/cakephp/cakephp/pull/11889